Commands/SearchCommands.cs

```diff
@@ -56,4 +56,21 @@ public static void GiveItem(ChatCommandContext ctx, string search, int page = 1)
 			Core.LogException(e);
 		}
 	}
+
+
+	[Command("List All Items", "LAI", adminOnly: true)]
+	public static void listAllItems(ChatCommandContext ctx) {
+		var gameDataSystem = Core.Server.GetExistingSystem<GameDataSystem>();
+		var managed = gameDataSystem.ManagedDataRegistry;
+
+		foreach (var entry in gameDataSystem.ItemHashLookupMap) {
+			try {
+				var item = managed.GetOrDefault<ManagedItemData>(entry.Key);
+				ctx.Reply(item.PrefabName);
```

Comment:
This is a lot of messages to send. I'd prefer for performance that it send fewer messages with newlines. I'm personally not sure that we should have both search and list items. It makes me think there's improvements that the search needs if it's not showing enough information.